styles.css

```diff
@@ -0,0 +1,148 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  display: flex;
+  justify-content: center;
+  background-color: #F9F9F9;
+  }
+
+.nav-bar {
+  display: inline-flex;
+  justify-content: space-between;
+  width: 100%;
+  position: top;
+  align-items: center;
+  }
+
+  #nav1 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  #nav2 {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+  }
+
+  #search-bar {
+    width: 700px;
+  }
+
+  #search-button {
+    width:10%;
+  }
+
+#img-logo {
+  height: 60%;
+  margin-right: 10em;
+  width: 100px;
+  margin-left: 15px;
+}
+
+#create-img, #apps-img, #notifications-img {
+  width: 50px;
+  height: 60%;
+}
+
+section {
+  width: 100;
+}
+
+h1 {
+  font-size: 24px;
+  font-family: Roboto, Arial, sans-serif;
+  text-decoration: underline;
+}
+
+iframe {
+  width: 68%;
+  height: 400px;
+}
+
+ul {
+  width: 68%;
+  display: inline-flex;
+  align-items: center;
+  align-content: center;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  display: inline-flex;
+  align-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  padding: 4px;
+  font-size: 14px;
+}
```

Comment:
You have some repeated code in these blocks. Always try to keep your code DRY. Please try to refactor these blocks.